jbpm-wb-human-tasks/jbpm-wb-human-tasks-client/src/main/java/org/jbpm/workbench/ht/client/editors/taskslist/AbstractTaskListBasicFiltersPresenter.java

```diff
@@ -94,6 +94,8 @@ public void loadFilters() {
 
         addProcessNameFilter(getDataSetId());
 
+        addContainerIdFilter(getDataSetId());
```

Comment:
@bxf12315 Container Id filter is rather technical, it shouldn't be added to the Task Inbox, only to Manage Tasks. Please add this logic to _TaskAdminListBasicFiltersPresenter_ only.

Reply:
just in admin, user can see the filter?